storage/grpc_client.go

```diff
@@ -522,7 +521,7 @@ func (c *grpcStorageClient) UpdateObject(ctx context.Context, bucket, object str
 	}
 	// Note: This API currently does not support entites using project ID.
 	// Use project numbers in ACL entities. Pending b/233617896.
-	if uattrs.ACL != nil {
+	if uattrs.ACL != nil || len(uattrs.PredefinedACL) > 0 {
```

Comment:
I'm surprised that this is necessary for PredefinedACL given how it is set in L485

Reply:
This line is to make sure the update mask contains something so we can send the request